next.config.js

```diff
@@ -14,14 +14,6 @@ module.exports = withBundleAnalyzer({
       };
     }
 
-    config.externals = [
```

Comment:
Is it worth leaving this commented as I think we should have another look at this at some point? 2mb is a big saving